python/ray/ml/examples/huggingface/huggingface_text_classification.ipynb

```diff
@@ -0,0 +1,1659 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "This notebook is based on [an official 🤗 notebook - \"How to fine-tune a model on text classification\"](https://github.com/huggingface/notebooks/blob/6ca682955173cc9d36ffa431ddda505a048cbe80/examples/text_classification.ipynb). The main aim of this notebook is to show the process of conversion from vanilla 🤗 to Ray AIR 🤗 without changing the training logic unless necessary."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "If you do not have 🤗 Datasets and 🤗 Transformers installed locally, uncomment and run the following line:"
```

Comment:
I really like this tutorial! We should sprinkle in a couple of references/links, e.g. for Ray Tasks, runtime envs, Ray Data or AIR itself. Apart from not being a huge fan of the `trainer_init_per_worker` name, I think this makes for a relatively smooth workflow. 👍